src/python/pants/backend/java/test/junit_test.py

```diff
@@ -55,7 +55,11 @@ def rule_runner() -> RuleRunner:
             QueryRule(TestResult, (JavaTestFieldSet,)),
         ],
         target_types=[JvmDependencyLockfile, JvmArtifact, JavaLibrary, JunitTests],
-        bootstrap_args=["--javac-jdk=system"],  # TODO(#12293): use a fixed JDK version.
+        bootstrap_args=[
+            "--javac-jdk=system",  # TODO(#12293): use a fixed JDK version.
+            # Makes JUnit output predictable and parseable across versions (#12933):
+            "--junit-args=['--disable-ansi-colors','--details=flat','--details-theme=ascii']",
```

Comment:
FYI this can all be a single string. The shell_str type will shlex things into a list for you.

Reply:
Neither of these appear to work:

> `            "--junit-args='--disable-ansi-colors --details=flat --details-theme=ascii'",`
or
> `             "--junit-args=\"--disable-ansi-colors --details=flat --details-theme=ascii\"",`

Maybe it's different when it comes from `RuleRunner`?

Reply:
What about the first one but without quotes?

Reply:
That works